mir_eval/hierarchy.py

```diff
@@ -143,6 +169,69 @@ def _lca(intervals_hier, frame_size):
     return lca_matrix.tocsr()
 
 
+def _meet(intervals_hier, labels_hier, frame_size):
+    '''Compute the (sparse) least-common-ancestor (LCA) matrix for a
+    hierarchical segmentation.
+
+    For any pair of frames ``(s, t)``, the LCA is the deepest level in
+    the hierarchy such that ``(s, t)`` are contained within a single
```

Comment:
I don't think that matters? All the comparisons are relative, so starting at 1 or 0 won't make a difference.